src/pages/workspaces/workspace/Tools.js

```diff
@@ -115,22 +120,35 @@ const ToolCard = pure(({ listView, name, namespace, config, onCopy, onDelete })
     style: styles.outerLink
   })
 
+  const redactedWarning = h(TooltipTrigger, {
+    content: 'This method is redacted'
+  }, [icon('warning', { size: 22, style: { color: colors.orange[0], ...styles.innerLink } })])
```

Comment:
```suggestion
  }, [icon('ban', { size: 22, style: { color: colors.orange[0], ...styles.innerLink } })])
```
"ban" icon is a circle with a cross-through, which I think sends a clearer message that the tool is permanently unavailable.